src/spec/integration/inventories-sync.spec.coffee

```diff
@@ -103,6 +103,75 @@ describe 'Integration Inventories Sync', ->
     .catch (error) -> done(_.prettify(error))
   , 60000
 
+  it 'should add restockableInDays', (done) ->
+    ie =
+      sku: 'xyz1'
+      quantityOnStock: 10
+      version: 1
+    ieChanged =
+      sku: 'xyz1'
+      quantityOnStock: 10
+      restockableInDays: 30
+      version: 1
+    @client.inventoryEntries.create(ie)
+    .then (result) =>
+      expect(result.statusCode).toBe 201
+      syncedActions = @sync.buildActions(ieChanged, result.body)
+      debug 'About to add to inventory with synced actions (restockableInDays add)'
+      @client.inventoryEntries.byId(syncedActions.getUpdateId()).update(syncedActions.getUpdatePayload())
+    .then (result) ->
+      expect(result.statusCode).toBe 200
+      expect(result.body.quantityOnStock).toBe 10
+      expect(result.body.restockableInDays).toBe 30
+      done()
+    .catch (error) -> done(_.prettify(error))
+
+  it 'should update restockableInDays', (done) ->
+    ie =
+      sku: 'xyz2'
+      quantityOnStock: 3
+      restockableInDays: 30
+      version: 1
+    ieChanged =
+      sku: 'xyz2'
+      quantityOnStock: 3
+      restockableInDays: 7
+      version: 1
+    @client.inventoryEntries.create(ie)
+    .then (result) =>
+      expect(result.statusCode).toBe 201
```

Comment:
same here, it would make it easier to follow IMO if we also added
`expect(result.body.restockableInDays).toBe 30`